src/identity/components/Name.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import { Name } from './Name';
+
+const meta = {
+  title: 'Identity/Name',
+  component: Name,
+  decorators: [
+    (Story) => {
+      return (
+        <QueryClientProvider client={new QueryClient()}>
+          <Story />
+        </QueryClientProvider>
+      );
+    },
+  ],
+  parameters: {
+    layout: 'centered',
+  },
```

Comment:
Let's move this to `.storybook/preview.ts`? Looks like it is shared among all stories.

Reply:
Good call. Moved all of them inside `.storybook/preview.ts`